### ISSUE TITLE
Crashes when right clicking

### ISSUE THREAD

**Issue by TyrusJames** [davqvist/Restriction#5]
I made chance cubes restricted to the hunting dimension, it says the tooltip on the cube, but game crashes when i place it.

**Comment by davqvist** [comment]
Please provide MC version, the json and crash log, maybe I can find the error. 
But if it is a bug in my code, I won't work on the mod anymore. Use https://www.curseforge.com/minecraft/mc-mods/restriction-re-restricted instead for 1.16 if possible

**Comment by TyrusJames** [comment]
Actually game crashes if i right click anytime. I'll find the files when i can get on pc.

**Comment by TyrusJames** [comment]
It's 1.12.2 using my modpack https://www.curseforge.com/minecraft/modpacks/sandwichcraft

This is the config I used and the crash report
[restriction.txt](https://github.com/davqvist/Restriction/files/7598461/restriction.txt)


[crash-2021-11-23_21.48.58-client.txt](https://github.com/davqvist/Restriction/files/7598459/crash-2021-11-23_21.48.58-client.txt)

**Comment by TyrusJames** [comment]
If i use the default config its fine and the potion stand is restricted from nether. When i try my version above to restrict chance cubes from hunting dimension, or the nether i tried too, it makes game crash on any right click action

**Comment by davqvist** [comment]
Can you try by turning ignoreMeta off?

**Comment by TyrusJames** [comment]
I tried with just chance cubes, and they work for normal and ico chance cubes, but giant ones still spawn as they create a massive block in the world... not sure it can block that... i think this issue is with this
{
      "block": "customnpcs:npcsoulstonefilled",
      "meta": 0,
      "ignoreMeta": true,
      "restrictions": [
        {
          "type": "DIMENSION",
          "reverse": false,
          "ignoreMeta": false,
          "id": 28885
        }
      ]
    }
    
I tried it with  "ignoreMeta": true, and false and still crashes it.  It doesn't like this soulstone from customnpcs - which is what I need this for mainly! :(
[crash-2021-11-25_10.49.00-client.txt](https://github.com/davqvist/Restriction/files/7602316/crash-2021-11-25_10.49.00-client.txt)

**Comment by davqvist** [comment]
Yeah no idea if multiblocks work properly.

**Comment by TyrusJames** [comment]
yeah, the main thing that's breaking it is customnpcs:npcsoulstonefilled  this is a soulstone that you grab an NPC with and place it again after. like the golden lasso... this is what is crashing it. Any thoughts on a way to do it? if not I'm not sure what to do.
